ITISFoundation/osparc-simcore
# Is921/study persistent data

## What do these changes do?

- [x] implements project clone => clones (with different ids) document + stored data (i.e. state + i/o files)
- [x] implements delete project => deletes document+stored data +   stops running services
- **extra**: Added remote debug option in development more for storage and webserver
- **extra**: fixes #954


## Related issue number
Implementing #921 
Implements #767
Fixes #954


## How to test

- login and add datcore keys
- create a project *that contains input and state data* (e.g. file-picker, computational service and a jupyter)
- save 
   - check project data folders in minio
- save as template
   - check project data folders in minio
- delete
   - check project data folders in minio
- create [rpekct from template
   - check project data folders in minio
## Checklist
